### PR TITLE
Add function messagesTaggedBy

### PR DESCRIPTION
**Pull request by wittjosiah**

### CODE REVIEW

**Comment by mixmix** [comment]
I think I would replace this with an `ssb-query` async version, but if this obs route is working ok keep using it.

**Comment by wittjosiah** [comment]
I'm going to merge this for now and it can be reviewed as necessary

File: pull.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+          }
+          if (!result[tagMsg.author]) return result
+          delete result[tagMsg.author][tagMsg.tag]
+          return result
+        }
+      }, {})
+      const count = reduce(tagDict, (result, authorTags) => result + size(authorTags), 0)
+      return count > 0
+    }),
+    pull.map((message) => message.message)
```
Comment (mixmix) [inline pull.js]: this is a little confusing. I think what's being mapped and filtered is not a message, it's an ... `item` being generated by that reduce I guess.

**Comment by mixmix** [comment]
@wittjosiah this is sweet, it feels really concise and looks like a good use of ssb-query

**Pull request merged**